frontend/packages/operator-lifecycle-manager/integration-tests/scenarios/global-installmode.scenario.ts

```diff
@@ -139,35 +139,10 @@ describe('Interacting with an `AllNamespaces` install mode Operator (Jaeger)', (
     expect(crudView.rowForName(jaegerOperatorName).isDisplayed()).toBe(true);
   });
 
-  xit(
```

Comment:
Removing disabled test. This isn't really testing the UI, but the operator itself. I don't see a lot of value in keeping this test.